mcm/json_layer/request.py

```diff
@@ -3025,11 +3025,19 @@ def request_to_tasks(self, base, depend):
 
         self.logger.info('Validation info of %s:\n%s', self.get_attribute('prepid'), dumps(validation_info, indent=2, sort_keys=True))
         filter_efficiency = self.get_efficiency()
+        request_skip_check = self.get_allowed_skip_validation_check().get(self.get_attribute("prepid"))
+        filter_eff_skipped = False
+        if request_skip_check:
+            # If any of the following options is set
+            skipped_if = set(("filter_efficiency", "all"))
+            if skipped_if.intersection(set(request_skip_check)):
+                filter_eff_skipped = True
+
         if validation_info:
             # Average filter efficiency
             filter_efficiency = sum([x['filter_efficiency'] for x in validation_info]) / len(validation_info)
             filter_efficiency_threshold = 0.001
-            if filter_efficiency < filter_efficiency_threshold:
+            if filter_efficiency < filter_efficiency_threshold and not filter_eff_skipped:
```

Comment:
`not filter_eff_skipped` is not in the right place here. 
this if is checking on the sheer value of the efficiency.